microservices/m2m-event-dispatcher/dev/values.yaml

```diff
@@ -0,0 +1,70 @@
+name: interop-be-m2m-event-dispatcher
+techStack: nodejs
+
+serviceAccount:
+  roleArn: "arn:aws:iam::505630707203:role/interop-be-m2m-event-dispatcher-dev-es1"
+
+service:
+  create: false
+
+configmap:
+  KAFKA_GROUP_ID: "{{.Values.namespace}}-m2m-event-dispatcher"
+  TOPIC_STARTING_OFFSET: "earliest"
+  M2M_EVENT_SCHEMA: "m2m_event" 
```

Comment:
since this is a shared configuration, any reason why it is not in `common-m2m-events-db` configmap?